src/mgr/BaseMgrModule.cc

```diff
@@ -676,6 +676,19 @@ get_unlabeled_perf_schema(BaseMgrModule *self, PyObject *args)
   return self->py_modules->get_unlabeled_perf_schema_python(type_str, svc_id);
 }
 
+static PyObject*
+get_perf_schema(BaseMgrModule *self, PyObject *args)
+{
+  char *type_str = nullptr;
+  char *svc_id = nullptr;
+  if (!PyArg_ParseTuple(args, "ss:get_perf_schema", &type_str,
+                                                    &svc_id)) {
+    return nullptr;
+  }
+
+  return self->py_modules->get_perf_schema_python(type_str, svc_id);
```

Comment:
where are the allocated type_str and svc_id deallocated?

Reply:
`type_str` and `svc_id` are pointers to string managed by Python, and python's garbage collector will take care of freeing the memory when it is no longer needed.

Ref: https://stackoverflow.com/questions/5606589/why-do-i-not-need-to-free-this-memory